src/ImageGallery.react.jsx

```diff
@@ -272,6 +272,7 @@ export default class ImageGallery extends React.Component {
               className={this.props.server ? 'loaded' : null}
               src={item.original}
               alt={item.originalAlt}
+              name={item.originalName}
```

Comment:
What do you need this for? This has been deprecated.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img